It was missed to update POWERTOP_VERSION macro so that the
new v2.2 release still shows v2.1 string on the header of
ncurses window.  Convert to use PACKAGE_VERSION macro which
set by autotools to the AC_INIT version so that it cannot
be missed anymore.

Signed-off-by: Namhyung Kim <namhyung at gmail.com>
---
 src/lib.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/lib.h b/src/lib.h
index 8cf4632..209421a 100644
--- a/src/lib.h
+++ b/src/lib.h
@@ -35,8 +35,8 @@
 
 #define _(STRING)    gettext(STRING)
 
-#define POWERTOP_VERSION "v2.1"
-#define POWERTOP_SHORT_VERSION "2.1"
+#define POWERTOP_VERSION "v"PACKAGE_VERSION
+#define POWERTOP_SHORT_VERSION PACKAGE_VERSION
 
 
 extern int get_max_cpu(void);
